style.css

```diff
@@ -0,0 +1,88 @@
+:root {
+    --white: white;
+    --black: black;
+    --gray: #d8d9db;
+    --gray-hoover: #ebebeb;
+    --gray-active: #bbbcbe;
+    --orange: #df974c;
+    --orange-hoover: #dfb07e;
+    --orange-active: #dd8d37;
+}
+
+/* Generics */
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: "Courier New", Courier, monospace;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+/* Calculator */
+
+.calc {
+    width: 400px;
+    background-color: var(--black);
+    color: var(--white);
+}
+
+.screen {
+    color: var(--white);
+    font-size: 40px;
+    text-align: right;
+    padding: 20px 5px;
+}
+
+.calc-row {
+    display: grid;
```

Comment:
I would expect the entire "calc" to be styled with "grid".

This is thhe correct way, to have just one element that is grid, instead of multiple like right now.

So, please refactor the code to have just one grid element, the one with the "calc" class.